### ISSUE TITLE
Privacy policy display

### ISSUE THREAD

**Issue by costafrancesco94** [orestbida/cookieconsent#307]
This is a wonderfoul project!!

I noticed there is no "standard" way to put the privacy policy link (in order to be fully compliant) except for hardcoding to languages translations (and place some html inside the strings)

Would be nice to have a policy link "space" (The famous `Read the complete privacy policy here`) and respective modal to be incuded in this project by default...

Some suggestion on how could be achieved (multiple way):
 - hardcoding policy in language file (bad, but easy)
 - retrieve policy through link (good, but require specific backend integration)

Will leave you some idea, if you feel this could be usefoul, I may prepare a PR
```
{
consent_modal: {},
settings_modal: {
    afterHeader: HTMLElement,
    beforeFooter: HTMLElement,
},
policy: {
    link: 'https://site.example.com/LANG-TAG/privacy'
    content: '<h1>Privacy Policy</h1><p>this document...</p><h2>Handling of Personal Data</h2><p>Personal......</p>...........'
    content_link: 'https://site.example.com/LANG-TAG/privacy?format=rawcontent'
}
```

**Comment by orestbida** [comment]
Hi @costafrancesco94,

Do you mean to implement something similiar to Iubenda's solution?

I don't understand why you need the `afterHeader` and `beforeFooter` options since there are already the blocks for this. Also, why are there 2 link options in the policy object?

**Comment by costafrancesco94** [comment]
> Do you mean to implement something similiar to Iubenda's solution?

I actually never used it, so i'm not sure about which part could be similar


> I don't understand why you need the `afterHeader` and `beforeFooter` options since there are already the blocks for this.
To not override your blocks. Those blocks could contain some other thing (such as text, icons, images) and custom part of the modal settings that could be more customised per project, while keeping the logic and the "main" aim of this project minimal and untouched

> Also, why are there 2 link options in the policy object?
They were ideas on how could that work

Mi idea is:
 - this program could also be able to display the policy without user change page
 - there are tons of way to do it, you should decide which ones are worth and usefoul

So, basically it could work in more way:
```
// main settings
{
// manage policy
 policy: {
  // option 1: load html string with the policy
    content: '<h1>Privacy</h1><p>Actual privacy policy</p>
  // option 2: load a htm element containing it
    content_html: HTMLElement
 // option 3: fetch it raw
   link: 'https://www.example.com/something/privacy'
// option 4: fetch a page and find a container ?
  page: 'https://www.example.com/something/privacy#privacyContainer
// option 5: ?? other ideas?
 }
}
```

Let me know if you are interested

**Comment by orestbida** [comment]
I assume there would be some kind of modal or tab in order to load/view the privacy policy content, which I'd rather not implement, just to keep things simpler.

**Comment by costafrancesco94** [comment]
I understand, i will found some solution that will stand beside your component, i will let you know how it works
